torch/csrc/jit/passes/freeze_module.cpp

```diff
@@ -1,8 +1,9 @@
-#include <torch/csrc/jit/passes/freeze_module.h>
 #include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/freeze_module.h>
 
 #include <torch/csrc/jit/ir/alias_analysis.h>
 #include <torch/csrc/jit/passes/inliner.h>
+#include <torch/csrc/jit/passes/quantization/helper.h>
```

Comment:
Can we remove quantanzation/helper.h dependency?
you are invoking findChildModule (small routine that can be inlined.